.github/workflows/eslint.yml

```diff
@@ -10,4 +10,6 @@ jobs:
       - name: install dependencies
         run: npm install
       - name: Run ESLint
-        run: npm run lint
\ No newline at end of file
+        run: npm run lint
+      - name: Run ESLint and Prettier
+        run: npm run lint-and-format
```

Comment:
The previous command was ok, you needed to fix the linting issues so that this would pass, we can have a 10 minute call sometime to finalise this